source/MainPanel.cpp

```diff
@@ -110,6 +110,32 @@ void MainPanel::Step()
 			isActive = !DoHelp("friendly disabled");
 		if(isActive && player.Ships().size() > 1)
 			isActive = !DoHelp("multiple ship controls");
+		if(isActive && flagship->HasBays())
+			isActive = !DoHelp("try out fighter fleet logistics");
```

Comment:
Not sure I follow, the hint only pops if you have a carrier flagship?

Reply:
In addition to escorts if you look at the full diff.  Flagship is tried first because that’s faster.

Reply:
Just so you know, some comments were just me asking for clarifications to be sure I got stuff right.
Your style of coding is new to me, and sometimes optimization is harder to understand.

Reply:
That's okay; I should favor clearer coding so I'm open to making changes for clarity which are not performance impacting.